### DISCUSSION TITLE
Help needed to setup locally

### DISCUSSION THREAD

**Discussion by hussamkhatib** [djyde/cusdis#212] [Q&A]
Hi there,
I want to contribute to this project and need help setting up local env.
I am not able to go past the sign in screen.
I have never used SQLite and have confusion If I am setting the path right.

This is my .env file (pasting as no secrets are exposed)
```
DB_URL=file:./db
USERNAME="admin"
PASSWORD="password"
JWT_SECRET="ofcourseistillloveyou"
NEXTAUTH_URL="http://localhost:3000"
```
I have ctreated the database and forked the repo in the same directory
![Screenshot from 2022-08-03 23-24-34](https://user-images.githubusercontent.com/52914487/182676940-f01ddaaa-ef2f-45c8-bd98-50c0bd811ef7.png)

**Comment by 12nick12** [comment]
I wasn't able to get mine to work with the .env file. I just created a systemd unit and pointed the EnvironmentFile to the env file.
